docs/en/connector-v2/source/Doris.md

```diff
@@ -0,0 +1,172 @@
+| fenodes                          | string | yes      | -          | FE address, the format is `"fe_host:fe_http_port"`                                                  |
+| username                         | string | yes      | -          | User username                                                                                       |
+| password                         | string | yes      | -          | User password                                                                                       |
+| database                         | string | yes      | -          | The name of Doris database                                                                          |
+| table                            | string | yes      | -          | The name of Doris table                                                                             |
+| schema                           | config | yes      | -          | The schema of the doris that you want to generate                                                   |
+| query-port                       | string | no       | 9030       | Doris QueryPort                                                                                     |
+| doris.filter.query               | string | no       | -          | Data filtering in doris. the format is "field = value".                                             |
+| doris.batch.size                 | int    | no       | 1024       | The maximum value that can be obtained by reading Doris BE once.                                    |
+| doris.request.query.timeout.s    | int    | no       | 3600       | Timeout period of Doris scan data, expressed in seconds.                                            |
+| doris.exec.mem.limit             | long   | no       | 2147483648 | Maximum memory that can be used by a single be scan request. The default memory is 2G (2147483648). |
+| doris.request.retries            | int    | no       | 3          | Number of retries to send requests to Doris FE.                                                     |
+| doris.request.read.timeout.ms    | int    | no       | 30000      |                                                                                                     |
+| doris.request.connect.timeout.ms | int    | no       | 30000      |                                                                                                     |
+
+### Tips
+
+> It is not recommended to modify advanced parameters at will
+
+## Task Example
+
+> This is an example of reading a Doris table and writing to Console.
+
+```
+env {
+  execution.parallelism = 2
+  job.mode = "BATCH"
+}
+source{
+  Doris {
+      fenodes = "doris_e2e:8030"
+      username = root
+      password = ""
+      database = "e2e_source"
+      table = "doris_e2e_table"
+      schema {
```

Comment:
We already have doris catalog, so I think we use catalog to get the fileds an data type. If we need let user config the fields they want to read, we can add a option `read_columns` to do this.